### PR TITLE
ignore .DS_Store class name

### PR DESCRIPTION
**Pull request by ruimashita**
ignore `.DS_Store` class name on Caltech format dataset

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: lmnet/lmnet/datasets/image_folder.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def classes(self):
         """Returns the classes list in the data set."""
 
         classes = os.listdir(self.data_dir)
+        classes = [class_name for class_name in classes if class_name != ".DS_Store"]
```
Comment (iizukak) [inline lmnet/lmnet/datasets/image_folder.py]: How about `classes .remove(".DS_Store"")` ?

Comment (ruimashita) [reply lmnet/lmnet/datasets/image_folder.py]: @iizukak 
`remove()` raise ValueError when there is no item.
https://docs.python.org/3/tutorial/datastructures.html

Comment (iizukak) [reply lmnet/lmnet/datasets/image_folder.py]: @ruimashita 
Should we care about no item case?

Anyway, I understood. It's ok to use list comprehension.

Comment (ruimashita) [reply lmnet/lmnet/datasets/image_folder.py]: @iizukak
my poor english...
No item means the `classes` don't include `.DS_Store`.
`remove()` raise ValueError when `classes` don't include `.DS_Store`.

```
>>> fruits = ["apple", "lemon"]
>>> fruits.remove("orange")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: list.remove(x): x not in list
```

Comment (iizukak) [reply lmnet/lmnet/datasets/image_folder.py]: Sorry, I miss read.

**Pull request merged**